src/components/molecules/Jukebox/Jukebox.scss

```diff
@@ -0,0 +1,93 @@
+@import "scss/constants.scss";
```

Comment:
please extract constants from literal values across this file

Reply:
fixed d0f3e28